tdrs-backend/setup.cfg

```diff
@@ -6,6 +6,7 @@ omit =
     tdpservice/settings/production.py
     tdpservice/settings/staging.py
     tdpservice/wsgi.py
+    tdpservice/scheduling/*
```

Comment:
I omitted `scheduling` to boost coverage, still not sure why it dropped 3% on my two line updates. Codecov says it it was related to files coming from develop that these commits didn't happen. Open to input on how to address this better.

<img width="1320" alt="Screenshot 2024-10-11 at 1 49 46 PM" src="https://github.com/user-attachments/assets/2881d057-485d-4711-934c-53b656590493">